packages/variable/webpack.config.js

```diff
@@ -0,0 +1,31 @@
+const TerserPlugin = require('terser-webpack-plugin');
+const path = require('path');
+
+module.exports = {
+  entry: ['./index.jsx'],
+  module: {
+    rules: [
+      {
+        test: /\.js(x?)$/,
+        use: {
+          loader: 'babel-loader',
+          options: {
+            extends: './.babelrc',
```

Comment:
You should be able to use the root Babel config instead

```suggestion
            extends: '../../.babelrc',
```